### PR TITLE
Release 1.15.0

### PR DESCRIPTION
**Pull request by haircommander**
Update deps and docs for release-15

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -52,7 +52,7 @@ RUN cd /tmp &&\
     mkdir -p ~/.parallel && touch ~/.parallel/will-cite
 
 # Install crictl and critest
-ENV CRICTL_COMMIT v1.14.0
+ENV CRICTL_COMMIT v1.15.0
```
Comment (saschagrunert) [inline Dockerfile]: This does not exist yet.

File: contrib/test/integration/main.yml [hunk 2]
```diff
@@ -19,13 +19,13 @@
     - name: clone build and install cri-tools
       include: "build/cri-tools.yml"
       vars:
-        cri_tools_git_version: "ff8d2e81baf8ff720fb916e42da57c2b772bd19e"
+        cri_tools_git_version: "v1.15.0"
```
Comment (saschagrunert) [inline contrib/test/integration/main.yml]: The same for this version.

File: Dockerfile (hunk 1 above)
Comment (haircommander) [reply Dockerfile]: hahah oops

**Comment by haircommander** [comment]
well, gnu's ftp servers are down, so gnu parallel can't be downloaded, so integration tests will just keep failing :)

**Comment by rhatdan** [comment]
/lgtm
@haircommander @saschagrunert @umohnani8 @mrunalp and everyone else involved in release
Great job
:1st_place_medal:

**Pull request merged**